.pipelines/1p-e2e.yml

```diff
@@ -16,27 +17,29 @@ resources:
         - repository: 1P
           type: git
           name: IDDP/msal-javascript-1p
-          ref: master
+          ref: update-validation-templates
```

Comment:
Is this intentional?

Reply:
Yes, it's temporary because the pipeline definitions aren't on master yet. I'll update before merging (builds will fail for that unfortunately)